### DISCUSSION TITLE
The table on the page is cropped. Link in the description.

### DISCUSSION THREAD

**Discussion by Mrpancho92** [mdn/yari#9890] [Q&A]
https://developer.mozilla.org/ru/docs/Web/JavaScript/Guide/Regular_expressions

If you substitute the value: max-width: 680px in the <td> tag, then everything is ok

**Comment by bsmth** [comment]
Thanks a lot for reporting, I'm closing this Discussion as I've created an issue where we can track this: https://github.com/mdn/yari/issues/9891

**Reply by caugner** [reply]
@Mrpancho92 See [my answer](https://github.com/mdn/yari/issues/9891#issuecomment-1784794525) in the issue for options to resolve this issue in the content.
